scico/linop/optics.py

```diff
@@ -351,60 +381,60 @@ def adequate_sampling(self):
 class FraunhoferPropagator(LinearOperator):
     r"""Fraunhofer (far-field) Propagator.
 
-    Notes:
+    Propagates a source field with coordinates :math:`(x_S, y_S)` to
+    a destination plane at a distance :math:`z` with coordinates
+    :math:`(x_D, y_D)`.
 
-        Propagates a source field with coordinates :math:`(x_S, y_S)` to
-        a destination plane at a distance :math:`z` with coordinates
-        :math:`(x_D, y_D)`.
+    The action of this linear operator is given by (Eq. 4.25,
+    :cite:`goodman-2005-fourier`)
 
-        The action of this linear operator is given by
-        (Eq 4.25, :cite:`goodman-2005-fourier`)
+    .. math ::
+        (A \mb{u})(x_D, y_D) = \underbrace{\frac{k_0}{2 \pi}
+        \frac{e^{j k_0 z}}{j z} \mathrm{exp} \left\{ j \frac{k_0}{2 z}
+        (x_D^2 + y_D^2) \right\}}_{\triangleq P(x_D, y_D)}
+        \int \mb{u}(x_S, y_S) e^{-j \frac{k_0}{z} (x_D x_S + y_D y_S)
+        } dx_S \ dy_S \;.
 
-        .. math ::
-            (A \mb{u})(x_D, y_D) = \underbrace{\frac{k_0}{2 \pi}
-            \frac{e^{j k_0 z}}{j z} \mathrm{exp} \left\{ j \frac{k_0}{2 z}
-            (x_D^2 + y_D^2) \right\}}_{\triangleq P(x_D, y_D)}
-            \int \mb{u}(x_S, y_S) e^{-j \frac{k_0}{z} (x_D x_S + y_D y_S)
-            } dx_S \ dy_S \;.
+    Writing the Fourier transform of the field :math:`\mb{u}` as
 
-        Writing the Fourier transform of the field :math:`\mb{u}` as
+    .. math ::
+        \hat{\mb{u}}(k_x, k_y) = \int e^{-j (k_x x + k_y y)}
+        \mb{u}(x, y) dx \ dy \;,
 
-        .. math ::
-            \hat{\mb{u}}(k_x, k_y) = \int e^{-j (k_x x + k_y y)}
-            \mb{u}(x, y) dx \ dy \;,
+    the action of this linear operator can be written
 
-        the action of this linear operator can be written
+    .. math ::
+        (A \mb{u})(x_D, y_D) = P(x_D, y_D) \ \hat{\mb{u}}
+        \left({\frac{k_0}{z} x_D, \frac{k_0}{z} y_D}\right) \;.
 
-        .. math ::
-            (A \mb{u})(x_D, y_D) = P(x_D, y_D) \ \hat{\mb{u}}
-            \left({\frac{k_0}{z} x_D, \frac{k_0}{z} y_D}\right) \;.
+    Ignoring multiplicative prefactors, the Fraunhofer propagated
+    field is the Fourier transform of the source field, evaluated at
+    coordinates :math:`(k_x, k_y) = (\frac{k_0}{z} x_D,
+    \frac{k_0}{z} y_D)`.
 
-        Ignoring multiplicative prefactors, the Fraunhofer propagated
-        field is the Fourier transform of the source field, evaluated at
-        coordinates :math:`(k_x, k_y) = (\frac{k_0}{z} x_D,
-        \frac{k_0}{z} y_D)`.
+    In general, the sampling intervals (and thus plane lengths)
+    differ between source and destination planes. In particular,
+    :cite:`voelz-2011-computational`
```

Comment:
@lukepfister: Do these equations correspond to Eq. 5.18 in `voelz-2011-computational`?

Reply:
yes

Reply:
Thanks. Added a note to the docs.